var/spack/repos/builtin/packages/rocm-debug-agent/package.py

```diff
@@ -71,6 +73,7 @@ class RocmDebugAgent(CMakePackage):
         "6.0.2",
         "6.1.0",
         "6.1.1",
+        "6.1.2",
```

Comment:
inside the rocm-debug-agent, can you combine the 2 for loops into one as the version list remains the same

Reply:
Removed redundant for loop